### PR TITLE
Migrate dasboot to require

### PR DESCRIPTION
**Pull request by aleksanb**
This is step 1 / N of transforming our code base so we can use the google closure compiler advanced settings.

### CODE REVIEW

File: nin/frontend/package.json [hunk 1]
```diff
@@ -25,8 +25,7 @@
     "less": "^2.7.2",
     "less-loader": "^3.0.0",
     "ng-annotate-loader": "^0.2.0",
-    "style-loader": "^0.14.0",
-    "webpack": "^2.2.1"
```
Comment (stianjensen) [inline nin/frontend/package.json]: I think removing webpack here introduces more issues than it solves

File: nin/dasBoot/bootstrap.js [hunk 2]
```diff
@@ -1,3 +1,53 @@
+window.THREE = require('./lib/00_three');
```
Comment (sigvef) [inline nin/dasBoot/bootstrap.js]: Can rename to not have numeric prefixes now, perhaps

Comment (aleksanb) [inline nin/dasBoot/bootstrap.js]: This comes in a later commit.

**Comment by aleksanb** [comment]
There is a performance hit on nin run of < 1 second.
Should be fine given that you're not nin running all the time.

**Comment by aleksanb** [comment]
Ping.

**Comment by stianjensen** [comment]
LGTM. @sigvef ?

**Comment by sigvef** [comment]
> There is a performance hit on nin run of < 1 second.
> Should be fine given that you're not nin running all the time.

I was thinking more of demo runtime performance. Does this impact running demos (especially when compiled?).

**Comment by sigvef** [comment]
I get this when running on `re`:

image

image

**Comment by aleksanb** [comment]
Don't think it should.
If anything, it would only impact load times,
as there's a call to __webpack_require to load a module now, as opposed to it being in global scope. But all the important variables are still bound to window, so playback performance should be the same.

There is a slight size increase in the demo (5kb?) but with future advanced optimizations the total size should go way down.

**Pull request merged**